backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -57,4 +60,49 @@ object RuntimeAttributesValidation {
     }
   }
 
+  def validateMemory(value: Option[WdlValue], onMissingKey: => ErrorOr[MemorySize]): ErrorOr[MemorySize] = {
+    value match {
+      case Some(i: WdlInteger) => parseMemoryInteger(i)
+      case Some(s: WdlString) => parseMemoryString(s)
+      case Some(_) => String.format(MemoryWrongFormatMsg, "Not supported WDL type value").failureNel
+      case None => onMissingKey
+    }
+  }
+
+  def parseMemoryString(s: WdlString): ErrorOr[MemorySize] = {
+    MemorySize.parse(s.valueString) match {
+      case scala.util.Success(x: MemorySize) =>
+        if (x.amount <= 0) {
```

Comment:
This is probably copy/paste from elsewhere but scala style guy suggests:

    val news = if (foo)
      goodNews()
    else
      badNews()